src/ansiblelint/rules/unnamed_task.py

```diff
@@ -1,48 +0,0 @@
-"""Implementation of unnamed-task rule."""
-
-# Copyright (c) 2016 Will Thames <will@thames.id.au>
```

Comment:
Do we need to include the copyright notice in the new file?

Reply:
Not really needed as it does not have a legal value, also the new one is practically a rewrite, that is why I did not include it.